spec/support/helpers.rb

```diff
@@ -0,0 +1,9 @@
+module Helpers
```

Comment:
Can we rename this to something like `JSONHelpers`? I've seen generically named "helper" classes bloat to 800+ lines ⛱